demo/src/index.html

```diff
@@ -84,32 +85,27 @@ <h2>Examples</h2>
               </ul>
             </section>
             <section class="section--uploader">
-              <div class="model-type-selector-container" style="float: right">
-                <h4>Type of model</h4>
-                <p>
-                  The <a href="https://github.com/elifesciences/sciencebeam-grobid-biorxiv">bioRxiv model</a>
-                  may work better on preprints and authors' original papers.</br/>
-                  Choose the "Other" model for publisher typeset papers.
-                </p>
-                <p class="model-type-selector">
-                  <button
-                    class="button button--default"
-                    id="biorxiv-model-button"
-                    value="biorxiv"
-                  >bioRxiv</button>
-                  <button
-                    class="button button--outline"
-                    id="other-model-button"
-                    value="other"
-                  >other</button>
-                </p>                              
+              <div class="model-type-selector-container">
+                <h2>Or upload a file</h2>
+                <p>Upload a DOC, DOCX or PDF file to view the converted XML version below. The bioRxiv model may work better on preprints and authors' original papers. Choose the "Other" model for publisher typeset papers.</p>
+
+                <div class="model-type-selector">
+                  <div for="type-of-model" class="type-of-model__label" aria-label="BioRxiv">bioRxiv</div>
+
+                  <label class="switch">
+                    <input type="checkbox" id="type-of-model" class="button type-of-model" value="biorxiv">
```

Comment:
The model selection seem to work. Although the `value` doesn't seem to change. Not sure if it matters though.